hostprocess/eventflow-logger/containerd-logger.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: containerd-logger
+  name: containerd-logger
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: containerd-logger
+  template:
+    metadata:
+      labels:
+        k8s-app: containerd-logger
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      containers:
+      - image: ghcr.io/kubernetes-sigs/sig-windows/eventflow-logger:v0.1.0
+        args: [ "config.json" ]
+        name: containerd-logger
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: containerd-logger-config
+          mountPath: /config.json
+          subPath: config.json
+      nodeSelector:
+        kubernetes.io/os: windows
+      tolerations:
```

Comment:
Should this be critical?

Reply:
cluster operates just fine without the logger

Reply:
I miss read this comment, I have removed this toleration